satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -73,11 +81,44 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 					Secret: &graphql.ArgumentConfig{
 						Type: graphql.NewNonNull(graphql.String),
 					},
+					CurrentReward: &graphql.ArgumentConfig{
+						Type: types.rewardInput,
+					},
+					FieldPartnerID: &graphql.ArgumentConfig{
+						Type: graphql.String,
+					},
+					ReferrerID: &graphql.ArgumentConfig{
+						Type: graphql.String,
+					},
 				},
 				// creates user and company from input params and returns userID if succeed
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
 					input, _ := p.Args[InputArg].(map[string]interface{})
 					secretInput, _ := p.Args[Secret].(string)
+
+					var (
+						partnerInput  string
+						referrerInput string
```

Comment:
How is this different than a partnerID ?

Reply:
I removed the logic for referrerID from this PR. But this should be the userID from referral link